test/connectivity/scripts/connectivityTest.sh

```diff
@@ -58,33 +68,73 @@ function prepare_test_from_artifacts() {
     
 }
 
-function print_logs() {
+function print_deployment_logs() {
+    print_highlighted_message 'LOGS FROM IOTEDGED'
+    journalctl -u iotedge -u docker --since "$test_start_time" --no-pager || true
+
+    print_highlighted_message 'EDGE AGENT LOGS'
```

Comment:
~We are potentially printing edge agent logs in two places.~ Also, the naming is inconsistent with ```edgeAgent LOGS```